module2/homework/transformContainers/test.cpp

```diff
@@ -1,6 +1,9 @@
 #include "gtest/gtest.h"
 
 // TODO: add proper includes
+#include <algorithm>
+
+#include "tranform.hpp"
 
 TEST(transformContainerTests, ShouldReturnUniqueMap) {
```

Comment:
Please think about adding one more test, where list and deque without duplicates have different amount of elements.
IMO in such case function should just return empty map.